include/kodlab_mjbots_sdk/mjbots_robot_interface.h

```diff
@@ -31,25 +31,23 @@ struct RealtimeParams {
   int lcm_cpu = 0;
 };
 
-class MjbotsRobotInterface {
+/*!
+ * @brief A class that allows interaction with the Mjbots Moteus Motor Controllers
+ * 
+ */
+class MjbotsRobotInterface  {
```

Comment:
Lets rename to mjbots hardware interface.

Reply:
This is covered in part 3/3 of the PR